### PR TITLE
add config for ASRock A300M-STX (DeskMini A300)

### PR DESCRIPTION
**Pull request by dbf**

### CODE REVIEW

File: configs/ASRock/A300M-STX-DeskMini.conf [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+
+   # fans
+   label fan2 "CPU_FAN1"
+   set fan2_min 500
+
+   label fan1 "CPU_FAN2"
+   set fan1_min 500
+   # it's near impossible to fit a 2nd fan (besides the CPU fan)
+   # in the DeskMini case, so ignore unconnected CPU_FAN2
+   ignore fan1
```
Comment (olysonek) [inline configs/ASRock/A300M-STX-DeskMini.conf:84]: That's funny :). Even in this case, I think I'd prefer not to ignore the fan in the upstream config.

File: configs/ASRock/A300M-STX-DeskMini.conf [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+
+   # these don't have connectors
+   ignore  fan3
+   ignore  fan4
+   ignore  fan5
+
+   # intrusion
+   # the DeskMini A300 has an intrusion detection connector
+   # on board, but the case has no switch
```
Comment (olysonek) [inline configs/ASRock/A300M-STX-DeskMini.conf:93]: I think I'd still prefer not to ignore the intrusion sensor in the upstream config.

File: configs/ASRock/A300M-STX-DeskMini.conf (hunk 1 above)
Comment (dbf) [reply configs/ASRock/A300M-STX-DeskMini.conf]: agree, changed

File: configs/ASRock/A300M-STX-DeskMini.conf (hunk 2 above)
Comment (dbf) [reply configs/ASRock/A300M-STX-DeskMini.conf]: agree, changed

**Comment by olysonek** [comment]
Thanks!

**Pull request merged**